udata_front/theme/gouvfr/assets/js/components/search/OrganizationSearch/OrganizationSearch.vue

```diff
@@ -1,10 +1,11 @@
 <template>
-  <div :id="id" class="fr-col fr-search-bar wrap" role="search">
+  <div :id="id" class="fr-col fr-search-bar relative wrap" role="search">
     <label class="fr-label" :for="inputId" :id="labelId">
       {{ t('Search for data') }}
     </label>
+    <span class="fr-icon-search-line" aria-hidden="true"></span>
     <input
```

Comment:
l.21 show me a ts error because showAndSelectIfQuery is expecting `q` and not a mouseEvent